php/libraries/NDB_Caller.class.inc

```diff
@@ -92,9 +92,7 @@ class NDB_Caller
         $phpfile   = $base . "project/instruments/"
             . "NDB_BVL_Instrument_$test_name.class.inc";
         $linstfile = $base."project/instruments/$test_name.linst";
-        if ($this->existsAndRequire($phpfile)
-            || $test_name=='instrument_preview'
```

Comment:
Even if the instrument builder added the button back, this is wrong.. the body of the if statement goes through the code path for survey instruments, not normal instruments.